app/javascript/components/shared/Header.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import { Navbar, NavDropdown, Stack } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useAuth } from '../sessions/AuthContext';
+
+export default function Header() {
+  const currentUser = useAuth();
+
+  return (
+    <Navbar bg="light">
+      <Container>
+        <Navbar.Brand as={Link} to="/">
+          <img
```

Comment:
I think using the react bootstrap `Image` component is more adequate.

Reply:
I got this from React Bootstrap demo.

https://react-bootstrap.github.io/components/navbar/

Reply:
Yes, but there exists this [component ](https://react-bootstrap.netlify.app/components/images/#images) it gives some extra stuff like shape that may be usueful.